app/controllers/api/validations_controller.rb

```diff
@@ -1,9 +1,16 @@
 module Api
   class ValidationsController < ApiController
+    # rubocop:disable Metrics/MethodLength
     def prisoner
+      begin
+        date = Date.parse(params.fetch(:date_of_birth))
+      rescue ArgumentError
+        raise ParameterError, 'date_of_birth'
+      end
```

Comment:
I'd go further. I think the parameter check would be better off handled in the validation object if possible, otherwise the validation logic is split between the controller and the validation object.